2018-11-26-sip-minutes.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: sips
+title: SIP Meeting Minutes - November 1-3 2018
+
+partof: minutes
+---
+
+# Minutes
+
+The following agenda was distributed to attendees, please [click here](https://docs.google.com/spreadsheets/d/1JdTyAxqqKqXtiXnZBPTaesMrWk7lyTWna1VPL3N5KWo/edit?usp=sharing).
```

Comment:
Could this just be

```suggestion
The agenda was distributed to attendees is [here](https://docs.google.com/spreadsheets/d/1JdTyAxqqKqXtiXnZBPTaesMrWk7lyTWna1VPL3N5KWo/edit?usp=sharing).
```

?

Reply:
Thanks for the wonderful feedback @hrhino !